azext_iot/central/_help.py

```diff
@@ -136,6 +136,51 @@ def _load_central_devices_help():
                 az iot central app device registration-summary --app-id {appid}
         """
 
+    helps[
+        "iot central app device run-command"
+    ] = """
+            type: command
+            short-summary: Run a command on a device and view associated response. Does NOT monitor property updates that the command may perform.
+            long-summary: |
+                Note: payload should be nested under "request".
+                i.e. if your device expects the payload in a shape {"key": "value"}
+                payload should be {"request": {"key": "value"}}.
```

Comment:
nit: consider adding flexibility to support either style. There is precedence...for example in [edge deployments](https://github.com/Azure/azure-iot-cli-extension/blob/dev/azext_iot/_help.py#L784)

Reply:
So I'd prefer not to do sniffing if possible.

2 main reasons that we can drill into below.
1) If a developer plans to use CLI as testing for how API's work, I want CLI to behave as closely to API as possible
2) No restrictions on param naming, so if a customer sets `request` as a param to a function, sniffing will lead to anomalous behavior

Drill down: 
1) A customer might just be playing around with CLI to expose concepts/test payloads. If they give the CLI a payload that doesn't work with the API, then they will be confused why the "api isn't working" when we have done sniffing for them.

2) The sniffing can also introduce ambiguity, which I'm not a personal fan of.

For example - there is no restriction for parameter names. So a user could have "request" as a perfectly valid name, which leads to "odd" behavior.

I.e.

```
{
  request: {
    request: ....
    a: ....
    ....
  }
}
```

But if the customer does something like:

```
{
  request: ....
  a: ....
  ....
}
```

Does request get passed to the device? are the other props ignored? etc

Reply:
Fair points particularly #2 resonates. There are more restrictions/structured format on edge payloads which doesn't have this ambiguity. For vanilla IoT Hub only the command payload is expected i.e. not {payload: {a: ..., b: ...}} but {a: ..., b: ...}.

I would caution the second part of #1 as a primary design goal as the CLI's purpose is not to mirror service API's...its closer to enabling users and scenarios to achieve Azure IoT goals through automation as well as supporting inner dev loop tooling. DevEx and enablement are among core goals. Users can always use debug facilities along with nicely layed out logging to understand more of how a command works.

Reply:
Ok that makes sense, yeah I'll keep your feedback on #1 in mind. If we get customer feedback that they want sniffing then I'll add more significant sniffing than just `request` sniffing. I'll likely do some `component_name` sniffing etc too

Reply:
for now #wontfix